src/components/CalendarModal/index.tsx

```diff
@@ -11,13 +13,19 @@ interface CalendarModalProps {
 export const CalendarModal: React.FC<CalendarModalProps> = ({ show, event, onClose }) => (
   <dialog className={s.modal} open={show}>
     <section className={s["modal-main"]}>
-      <h3>What type of calendar event would you like?</h3>
+      <h3>Add Event to Calendar</h3>
       <div className={s.options}>
-        <button onClick={() => saveToAppleCal(event)}>Apple Calendar</button>
-        <button onClick={() => saveToGoogleCal(event)}>Google Calendar</button>
+        <button className={s.google} onClick={() => saveToGoogleCal(event)}>
```

Comment:
Let's keep the className `google` and `apple` unchanged since they are defining the unique background colors for each button but add a second className like `calendarButton` for all of the other attributes that are shared in common between both buttons.

The syntax might look a little funny but we need to wrap the classes in a template literal string to include multiple classes so it would look like this.
```suggestion
        <button className={`${s.google} ${s.calendarButton}`} onClick={() => saveToGoogleCal(event)}>
```